protocol/dialer.go

```diff
@@ -170,6 +184,12 @@ func attemptFetch(ctx context.Context, nonTlsConn net.Conn, creds *types.NodeCre
 		return nil, fmt.Errorf("(%s) error splitting request into next protos: %w", op, err)
 	}
 
+	// Add the common dns name certificate selector. We don't have anything
+	// better to use here because we don't have certs, and on the other side
+	// this is an indicator that it's a fetch request.
+	// splitNextProtos = append(splitNextProtos,
```

Comment:
Did you mean to leave this commented out?

Reply:
Ah, that can be removed, thanks.